zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -223,6 +236,27 @@ def render_events(self, events_dict: Dict[str, Any]) -> List[str]:
             example = json.dumps(events["example"], indent=4, sort_keys=True)
             text.append(example)
             text.append("```\n\n")
+
+        # Creates table of links for events
+        events_table_str: List[str] = ['<div class="events-table">']
+        sorted_events_table: OrderedDict[str, List[str]] = OrderedDict(sorted(events_table.items()))
+        for event_name, ops in sorted_events_table.items():
+            if not ops:
+                events_table_str.append(
+                    table_link_template.format(link_name=event_name, url=event_name)
+                )
+            else:
+                events_table_str.append(f'<div class="events-table-link">{event_name}:</div>')
+                events_table_str.append("<div class='events-table-ops'>")
+                ops.sort()
+                op_list = [
+                    table_link_template.format(link_name=op, url=f"{event_name}-{op}") for op in ops
+                ]
+                events_table_str.extend(op_list)
+                events_table_str.append("</div>")
+        events_table_str.append("</div>")
```

Comment:
Is it easy to write this using a `Jinja2` template (i.e. the things under `templates/`) instead? Generally our policy is for HTML to be done in templates unless there's a very good reason to do something different.

Reply:
Might be something we should leave to a follow-up, since this file doesn't use those as a general pattern.